### PR TITLE
Shadowling balance #2

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
balance: Slings take less damage from red light
balance: Slings take slightly less damage from green light
tweak: You no longer need the special dethralling surgery to dethrall people, normal organ manipulation is fine
fix: Thrall darksight now works, after almost half the year!
fix: Ascendant Shadowlings Lightning Storm now actually works
add: Ascendant Shadowlings now have the Gore ability, which allows them to instantly dash into and gore an enemy.
/:cl:

### CODE REVIEW

File: hippiestation/code/game/gamemodes/shadowling/shadowling.dm [hunk 1]
```diff
@@ -83,6 +87,7 @@ Made by Xhuis
 	recommended_enemies = 2
 	restricted_jobs = list("AI", "Cyborg")
 	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	enemy_minimum_age = 14
```
Comment (Carbonhell) [inline hippiestation/code/game/gamemodes/shadowling/shadowling.dm]: changelog this change

Comment (ghost) [reply hippiestation/code/game/gamemodes/shadowling/shadowling.dm]: Done

**Comment by ghost** [comment]
@HippieStation/head did you guys give this +1 or +2  already?
@HippieStation/maintainers review please

**Comment by ghost** [comment]
we gave it +0 because of whitelist

**Comment by ghost** [comment]
@Dudeinyourbackyard there IS NO WHITELIST.

**Comment by ghost** [comment]
Why the fuck would you think theres a whitelist? I just made it so you have to had played on the server for 2 or more weeks to unlock slings, like you do with clock cult, blood cult, wizard, nukeops, etc

**Comment by ghost** [comment]
@Dudeinyourbackyard so you still keeping the +0? because there is no "whitelist".

**Comment by ghost** [comment]
@HippieStation/head Please, +1 this before git history gets fucked again

**Comment by ghost** [comment]
:/ this is becoming a needlessly long wait

**Comment by ghost** [comment]
uhh not whitelist moreso wait time
das what i been meaning

**Comment by ghost** [comment]
Ok fine i'll remove it.

**Comment by ghost** [comment]
@Dudeinyourbackyard wait time removed....

**Comment by ghost** [comment]
+1

File: hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 2]
```diff
@@ -851,14 +880,14 @@
...
-			if(is_shadow_or_thrall(target))
-				continue
-			to_chat(target, "<span class='userdanger'>You are struck by a bolt of lightning!</span>")
-			playsound(target, 'sound/magic/LightningShock.ogg', 50, 1)
-			target.Knockdown(8)
-			user.Beam(target,icon_state="red_lightning",icon='icons/effects/effects.dmi',time=1)
+	for(var/mob/living/carbon/human/target in view(6))
+		if(is_shadow_or_thrall(target))
+			continue
+		to_chat(target, "<span class='userdanger'>You	 are struck by a bolt of lightning!</span>")
```
Comment (ghost) [inline hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm]: > You [huge blob of whitespace here] are

Fix that

Comment (ghost) [reply hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm]: Done

**Comment by ghost** [comment]
sure +1

**Comment by bardofrage** [comment]
+1 gibe me darksight

File: hippiestation/code/game/gamemodes/shadowling/ascendant_shadowling.dm [hunk 3]
```diff
@@ -32,18 +32,30 @@
...
 	for(var/mob/M in GLOB.mob_list)
 		to_chat(M, "<span class='notice'><i><b>You feel a woosh as newly released energy temporarily distorts space itself...</b></i></span>")
-		M.playsound_local(get_turf(src), 'sound/hallucinations/wail.ogg', 150, 1, pressure_affected = FALSE)
-	. = ..(args)
\ No newline at end of file
+		SEND_SOUND(M, sound('sound/hallucinations/wail.ogg'))
+	. = ..(args)
+
+
+/mob/living/simple_animal/ascendant_shadowling/with_spells
```
Comment (ghost) [inline hippiestation/code/game/gamemodes/shadowling/ascendant_shadowling.dm]: this is not needed

Comment (ghost) [reply hippiestation/code/game/gamemodes/shadowling/ascendant_shadowling.dm]: ohk, i just used that during testing.

**Comment by MoreRobustThanYou** [comment]
build this

**Comment by ghost** [comment]
build this.

**Comment by ghost** [comment]
stupid bot

**Comment by praisenarsie** [comment]
what happened to travis

**Comment by PopNotes** [comment]
I can't believe jenkins is fucking dead

**Comment by praisenarsie** [comment]
yeah who removed travis?

**Review by ghost — APPROVED** [review]

**Comment by GuyonBroadway** [comment]
Approved.

**Pull request merged**